src/MockBot.Api/WeatherForecast.cs

```diff
@@ -0,0 +1,13 @@
+namespace MockBot.Api
```

Comment:
Perhaps a silly question, but why is this weather stuff in here? I saw 'weather' before and I assumed it was something like Jenkins has sunny/cloudy/rainy icons for build successes. This is like real temperatures though?

Reply:
This is going to get removed actually. It is part of the default template which is why it is there, but I'm going to replace it with something else which I'll write some simple tests for to get us over the 80% threshold. I'll leave this comment one until I've done that.

Reply:
Have now removed weather code.